core/crypto/node_ks.go

```diff
@@ -149,6 +149,10 @@ func (ks *keyStore) storePrivateKeyInClear(alias string, privateKey interface{})
 	return nil
 }
 
+func (ks *keyStore) deletePrivateKeyInClear(alias string) error {
+	return os.Remove(ks.node.conf.getPathForAlias(alias))
```

Comment:
thanks @srderson
The user is not able to alter those values as their inputs are statically specified in yaml files (e.g. TLSCertFilename and TLSKeyFilename). Besides that, I think your point is worth to be addressed in a PR to cover the entire project.